### PR TITLE
RavenDB-11140 backup now studio work

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Server/Web/System/OngoingTasksHandler.cs [hunk 1]
```diff
@@ -532,7 +532,7 @@ public async Task BackupDatabaseOnce()
 
                     var t = Database.Operations.AddOperation(
                         null,
-                        $"Database backup: {Database.Name}",
+                        $"Manual backup for database: {Database.Name}",
```
Comment (ml054) [inline src/Raven.Server/Web/System/OngoingTasksHandler.cs:535]: Maybe `Manual database backup` ?

File: src/Raven.Studio/typescript/common/appUrl.ts [hunk 2]
```diff
@@ -184,9 +185,12 @@ class appUrl {
...
 
-    static forDatabases(databaseToCompact?: string): string {
-        const compactPart = databaseToCompact ? "?compact=" + encodeURIComponent(databaseToCompact) : "";
-        return "#databases" + compactPart;
+    static forDatabases(databaseToCompact?: string, restore?: boolean): string {
+        const paramPart = databaseToCompact || restore ? "?" : "";
+        const compactPart = databaseToCompact ? "compact=" + encodeURIComponent(databaseToCompact) : "";
+        const andPart = databaseToCompact && restore ? "&" : "";
+        const restorePart = restore ? "restore=true" : "";
+        return "#databases" + paramPart + compactPart + andPart + restorePart;
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/appUrl.ts]: maybe something like:
```
const tokens: string[] = [];
  if (databaseToCompact) {
     tokens.push("compact=" + encodeURIComponent(databaseToCompact));
  }
  if (restore) {
     tokens.push("restore=true");
  }
  
  const mergedTokens = tokens.join("&");
  
  return "#databases" + (mergedTokens ? "?" + mergedTokens : "");
```

File: src/Raven.Studio/typescript/common/shell/menu/items/settings.ts [hunk 3]
```diff
@@ -163,7 +163,7 @@ function getSettingsMenuItem(appUrls: computedAppUrls) {
             title: 'Backup Task',
             nav: false,
             dynamicHash: appUrls.editExternalReplicationTaskUrl,
-            itemRouteToHighlight: 'databases/tasks/ongoingTasks'
+            itemRouteToHighlight: 'databases/tasks/backups'
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/shell/menu/items/settings.ts:166]: title : Backup Task dynamic hash: ExternalReplication, itemRouteToHighlight: backup. This is suspicious.

File: src/Raven.Studio/typescript/models/database/tasks/periodicBackup/manualBackupConfiguration.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                 serverLimits: periodicBackupServerLimitsResponse,
+                 encryptedDatabase: boolean) : manualBackupConfiguration {
+        return new manualBackupConfiguration(databaseName, backupConfiguration.emptyDto(), serverLimits, encryptedDatabase);
+    }
+    
+    submit(db: database, cfg: Raven.Client.Documents.Operations.Backups.BackupConfiguration) {
+        return new backupNowManualCommand(db, cfg)
+            .execute()
+            .done((manualBackupResult: Raven.Client.Documents.Operations.Backups.StartBackupOperationResult) => {
+                if (manualBackupResult && manualBackupResult.OperationId) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/tasks/periodicBackup/manualBackupConfiguration.ts]: do we need this if here?

File: src/Raven.Server/Web/System/OngoingTasksHandler.cs (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Server/Web/System/OngoingTasksHandler.cs:535]: Currently, it looks like this:
image

imho - it makes more sense to have:
_Manual backup for database: q1_

do you still prefer:
_Manual database backup: q1_
?

File: src/Raven.Studio/typescript/common/shell/menu/items/settings.ts (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/common/shell/menu/items/settings.ts:166]: fixed

File: src/Raven.Studio/typescript/models/database/tasks/periodicBackup/manualBackupConfiguration.ts (hunk 4 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/tasks/periodicBackup/manualBackupConfiguration.ts]: removed

File: src/Raven.Studio/typescript/common/appUrl.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/common/appUrl.ts]: refactored

File: src/Raven.Server/Web/System/OngoingTasksHandler.cs (hunk 1 above)
Comment (ml054) [reply src/Raven.Server/Web/System/OngoingTasksHandler.cs:535]: ok let's leave it

**Review by ml054 — APPROVED** [review]

**Pull request merged**